addons/website/models/ir_module_module.py

```diff
@@ -152,9 +152,18 @@ def _update_records(self, model_name, website):
                 # special case for attachment
                 # if module B override attachment from dependence A, we update it
                 if not find and model_name == 'ir.attachment':
-                    find = rec.copy_ids.search([('key', '=', rec.key), ('website_id', '=', website.id)])
+                    find = rec.copy_ids.search([('key', '=', rec.key), ('website_id', '=', website.id), ("original_id", "=", False)])
 
                 if find:
+                    # XXX
+                    # There is no flow that leads to 'find' having more than one record.
+                    # However, a user can still make it happen by duplicating an attachment and
+                    # manually setting 'theme_template_id' on it.
```

Comment:
Let's not talk about that flow, we don't really support those backend tweak flow.
What we are fixing here is just the legit flows duplicating attachment (eg image CROP) that "broke" the DB  **before** the fix copy=False on theme_template_id was applied (even if the backend tweak flow will also be fixed by that, but that's not our goal).